release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -467,7 +467,32 @@ In the table, features are marked with the following statuses:
 === Removed features
 
 [id="ocp-4-6-bug-fixes"]
-== Bug fixes
+== Bug fixes 
+
+*Web console (Developer perspective)*
+
+* Previously, when you tried to delete a Knative application through the *Topology* view, a false positive error about a non-existing *Knative route* was reported. This issue is now fixed and the error is no longer displayed. link:https://bugzilla.redhat.com/show_bug.cgi?id=1866214[BZ#1866214])
+
+* Previously, Developer Console did not allow images from insecure registries to be imported. This bug fix adds a checkbox that allows user to use the insecure registries in the *Deploy image* form (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826740[BZ#1826740])
+
+* When a user selected the *From Catalog* option to to create an application, the *Developer Catalog* displayed a blank page instead of a list of templates to create an application. This was caused when the 1.18.0 Jaeger operator was installed. This issue has now been fixed and the templates are displayed as expected. link:https://bugzilla.redhat.com/show_bug.cgi?id=1845279[BZ#1845279])
+
+* When adding or deleting a task in a Pipeline through the *Pipeline Builder* in the Developer Console, the interface was rearranging the tasks incorrectly. With this fix the parallel tasks are reconnected with the connector on addition or deletion of a task and the positions of the tasks are not rearranged. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856155[BZ#1856155])
+
+* A web console was crashing with a JavaScript exception when the user cancelled the creation of a Pipeline through a web console. This was fixed by improving the internal state handling. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856267[BZ#1856267])
+
+* A user having the required permissions was unable to retrieve and deploy an image from another project. The required rolebindings have now been created to fix this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843222[BZ#1843222])
+
+* When you tried to deploy an application from a Git repository with *Import from Git* function, the Developer Console reported a false positive error `Git repository is not reachable` for private repositories reachable by the cluster. This was fixed by adding information on making the private repository available to the cluster in the error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1877739[BZ#1877739])
```

Comment:
s/with \*Import from Git\*/with the \*Import from Git\*

Reply:
```suggestion
* When you tried to deploy an application from a Git repository with the *Import from Git* function, the Developer Console reported a false positive error `Git repository is not reachable` for private repositories reachable by the cluster. This was fixed by adding information on making the private repository available to the cluster in the error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1877739[BZ#1877739])
```